01-intro.Rmd

```diff
@@ -37,24 +59,82 @@ The major requirement of the policy is that all grant proposals (submitted after
+- **Can I elect to not share data?** The short answer is yes. However, you must still justify this choice in a submitted DMS Plan.
+
+## Does my research generate scientific data?
+
+The NIH Data Management and Sharing (DMS) Policy applies to **all NIH-supported research generating scientific data**. But what is "scientific data"?
+
+**SCIENTIFIC DATA**
+
+Scientific data are the “recorded factual material of sufficient quality to validate and replicate research findings, regardless of whether the data are used to support scholarly publications”. This can include any of the following if they are applicable to your study:
+
+- unpublished results
+- null results
+- results used to publish papers
+<!-- TODO: are the following two points correct? -->
+- raw data
+- cleaned or processed data
+
+**NOT SCIENTIFIC DATA**
+
+You are not expected to share:
+
+- lab notebooks
+- preliminary analyses
+- case report forms
+- drafts of scientific papers
+- plans for future research
+- peer reviews
+- communications with colleagues
+- physical objects (such as biospecimens)
+
+### What grant mechanisms don't require compliance with the policy?
+
+<!-- Fill in here -->
+
+
+
+## What is the timeline for sharing data?
+
+Data should be made available **no later than publication or end of the award**. This means that data underlying findings that are not published in peer-reviewed journals should be made available by the end of the award.
+
+### What about no cost extensions?
+
+Scientific data should be made accessible as soon as possible, and no later than the time of an associated publication or the end of the performance period of the extramural award that generated the data. If a no cost extension is granted for an extramural award, scientific data should be made accessible no later than the time of an associated publication, or the end of the no cost extension, whichever comes first.
+
+### What about grant renewals? 
+
+<!-- Fill in here -->
+
+
+
+## Can I elect to not share data?
+
+There are legitimate reasons you might not want to share your data. Data might not be sharable due to ethical, legal, or technical concerns. **You will still need to submit a DMS Plan even if you plan to withold data sharing**. You must explain your reasoning in your DMS Plan.
+
+Justifiable ethical, legal, and technical factors for limiting sharing of data include: 
 
-What about no cost extensions?
+- Informed consent will not permit or will limit the scope or extent of sharing and future research use
+- Existing consent (e.g., for previously collected biospecimens) prohibits sharing or limits the scope or extent of sharing and future research use
+- Privacy or safety of research participants would be compromised or place them at greater risk of re-identification or suffering harm, and protective measures such as de-identification and [Certificates of Confidentiality](https://grants.nih.gov/policy/humansubjects/coc.htm) would be insufficient
```

Comment:
I think we should put a disclaimer about this- that we suggest they hire an expert to help de-identify the data - maybe we can point them to ideas of where to find such people too?

Reply:
definitely!